apps/studio/src/pages/sites/[siteId]/collections/[resourceId].tsx

```diff
@@ -42,16 +51,10 @@ const CollectionResourceListPage: NextPageWithLayout = () => {
   return (
     <>
       <DashboardLayout
-        breadcrumbs={[
-          {
-            href: `/sites/${siteId}`,
-            label: "Home",
-          },
-          {
-            href: `/sites/${siteId}/collections/${resourceId}`,
-            label: metadata.title,
-          },
-        ]}
```

Comment:
previously it was hardcoded because we assumed it will be root level, which is no longer true with this update